bottles/frontend/ui/details-preferences.ui

```diff
@@ -754,7 +754,7 @@
                     <object class="AdwActionRow">
                         <property name="activatable-widget">switch_obsvkc</property>
                         <property name="title" translatable="yes">OBS Vulkan Capture</property>
-                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all programs.</property>
+                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all Vulkan programs.</property>
```

Comment:
```suggestion
                        <property name="subtitle" translatable="yes">Toggle OBS game capture for all programs.</property>
```
I'm pretty sure the obs-vkcapture isn't only for vulkan

Reply:
You're right, it's for OpenGL too.

Reply:
Oh yes, that is correct. Based on [this repo](https://github.com/nowrep/obs-vkcapture), it functions for OpenGL applications as well.